### ISSUE TITLE
timelion fails to install on Kibana 4.2.1 on Linux

### ISSUE THREAD

**Issue by joshuar** [elastic/timelion#22]
After downloading and unpacking a fresh 4.2.1 64-bit Linux tar.gz of Kibana, I get the following error trying to install timelion:

```
kibana-4.2.1-linux-x64 $ bin/kibana plugin --install kibana/timelion
Installing timelion
Attempting to extract from https://download.elastic.co/kibana/timelion/timelion-latest.tar.gz
Downloading 3382409 bytes....................
Extraction complete
Optimizing and caching browser bundles...
Plugin installation was unsuccessful due to error "Optimizations failure.

    ERROR in ./installedPlugins/.plugin.installing/public/directives/chart_directive.js
    Module not found: Error: Cannot resolve module 'moment-timezone' in /home/joshua/Downloads/elastic/kibana-4.2.1-linux-x64/installedPlugins/.plugin.installing/public/directives
     @ ./installedPlugins/.plugin.installing/public/directives/chart_directive.js 5:13-39
```

**Comment by johtani** [comment]
I have same error in my Mac OS with Kibana 4.2.1 and 4.2.0.

**Comment by nxhack** [comment]
issue effect this:  Use browser time for display. Closes #19
